beep/structure.py

```diff
@@ -395,6 +396,14 @@ def get_interpolated_diagnostic_cycles(self, diagnostic_available,
 
         diag_data = self.data[self.data['cycle_index'].isin(diag_cycles_at)]
 
+        #Convert date_time_iso field into pd.datetime object
```

Comment:
use a hash+space for line-length comments

Hopefully we'll be able to automate PEP checks once we get travis, etc. going.